src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -508,7 +493,8 @@ public CaseDTO modifyCase(ModifyCaseRequestDTO modifyRequestDTO) throws CTPExcep
     UUID originalCaseId = modifyRequestDTO.getCaseId();
     UUID caseId = originalCaseId;
 
-    CaseContainerDTO caseDetails = getCaseFromRmOrCache(originalCaseId, false);
+    CaseContainerDTO caseDetails = getCaseFromRmOrCache(originalCaseId, true);
+    caseDetails.setCreatedDateTime(DateTimeUtil.nowUTC());
```

Comment:
Name createdDateTime becomes a bit of a misnomer used for modifications, becomes lastUpdatedDateTime.

Reply:
I agree. Phil and I discussed it, and maybe this could be improved in the future?